rxbinding-support-v4/src/main/java/com/jakewharton/rxbinding/support/v4/widget/RxSlidingPaneLayout.java

```diff
@@ -0,0 +1,70 @@
+package com.jakewharton.rxbinding.support.v4.widget;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.support.v4.widget.SlidingPaneLayout;
+import rx.Observable;
+import rx.functions.Action1;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkNotNull;
+
+/**
+ * Static factory methods for creating {@linkplain Observable observables}
+ * for {@link SlidingPaneLayout}
+ */
+public final class RxSlidingPaneLayout {
+  /**
+   * Create an observable of the open state of the pane of {@code view}
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link SlidingPaneLayout#setPanelSlideListener}
+   * to observe dismiss change. Only one observable can be used for a view at a time.
+   * <p>
+   * <em>Note:</em> A value will be emitted immediately on subscribe.
+   */
+  @CheckResult @NonNull public static Observable<Boolean> paneOpen(
```

Comment:
I think this should be named `panelOpens` and the other should be `panelSlides` to sick with the naming convention.